storage/innobase/xtrabackup/test/t/xb_duplicate_sdi.sh

```diff
@@ -0,0 +1,80 @@
+
+############################################################################
+# PXB-2101 --prepare crash if exchange partition happened
+############################################################################
+require_server_version_higher_than 8.0.22
```

Comment:
btw, why do we need server version > 8.0.22 ?

Reply:
also is it possible for a test in PXB (without changing bootstrap or other config files)
1. ask for a specific server version 8.0.22
2. and run the same test on the latest server version

Reply:
since we are running prepare against saved backup.

Reply:
Ok